lib/inc/facter/execution/execution.hpp

```diff
@@ -11,10 +11,16 @@
 #include <utility>
 #include <stdexcept>
 #include <functional>
-#include "../util/option_set.hpp"
-#include "../util/environment.hpp"
 #include "../export.h"
 
+#include <leatherman/util/environment.hpp>
+#include <leatherman/util/option_set.hpp>
+#include <leatherman/execution/execution.hpp>
+#include <leatherman/execution/execution.hpp>
+
+namespace lth_util = leatherman::util;
+namespace lth_exec = leatherman::execution;
+
 namespace facter { namespace execution {
 
     /**
```

Comment:
We preferably don't want to refer to `leatherman` in the inc/facter header files at all, which means we'll need a lot of interface classes. @peterhuene any thoughts on alternatives?